api/pkg/parser/catalog.go

```diff
@@ -264,6 +266,14 @@ func (c CatalogParser) appendVersion(res *Resource, filePath string) Result {
 		result.Info(issue)
 	}
 
+	// optional check
+	deprecated, ok := annotations[DeprecatedAnnotation]
+	if ok {
+		issue := fmt.Sprintf("Resource %s - %s version of %s has been deprecated", tkn.GVK, version, tkn.Name)
+		log.With("action", "ignore").Info(issue)
+		result.Info(issue)
```

Comment:
wouldn't this add to `catalogerror` table?

Reply:
do we need to add that record?

Reply:
not sure on that ,have added just for info